src/crud.js

```diff
@@ -0,0 +1,148 @@
+import dots from '../assets/images/dots.png';
+import deleteIcon from '../assets/images/delete.png';
+import saveIcon from '../assets/images/save.jpeg';
```

Comment:
[ ] - Please fix this issue I found in webpack 
![image](https://user-images.githubusercontent.com/74744787/184036886-ef363100-0e6b-42bd-8996-80658d9be1de.png)

Reply:
Okay thank you. Will do that straight away

Reply:
Done